react/__tests__/Backdrop.test.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { render, fireEvent } from '@vtex/test-tools/react'
+
+// eslint-disable-next-line jest/no-mocks-import
+import { findCSSHandles } from '../__mocks__/testUtils'
+import Backdrop, { CSS_HANDLES } from '../components/Backdrop'
+
+describe('<Backdrop />', () => {
+  it('should render', () => {
+    const { baseElement } = render(<Backdrop open />)
+
+    expect(baseElement).toBeTruthy()
+  })
+
+  it('should find all declared handles', () => {
+    const { container } = render(<Backdrop open />)
+
+    const foundHandles = findCSSHandles(container, CSS_HANDLES)
+    expect(foundHandles).toEqual(CSS_HANDLES)
```

Comment:
I think on this case it might be better to not export CSS_HANDLES in the backdrop component and then you create the variable here so it is more explicit what exactly is the expected result

Reply:
But what I want to test here is if we are using all the handles from `CSS_HANDLES` in the elements of the component